seed/static/seed/js/controllers/delete_column_modal_controller.js

```diff
@@ -24,10 +24,9 @@ angular.module('BE.seed.controller.delete_column_modal', [])
       $scope.delete = function () {
         $scope.state = 'pending';
         columns_service.delete_column_for_org(organization_id, column.id).then(function (result) {
-          $scope.state = 'running';
-          $scope.startTime = moment();
+          $scope.state = 'evaluating';
           $scope.interval = $interval(function () {
-            $scope.updateTime();
+            $scope.state == 'running' ? $scope.updateTime() : $scope.setRunningState();
```

Comment:
New state while the celery task is evaluating the queryset for length. If we're still evaluating there is no need to update the time